responses/test_matchers.py

```diff
@@ -283,17 +283,24 @@ def run():
     assert_reset()
 
 
-def test_multipart_matcher():
+@pytest.mark.parametrize("match_file", (b"Old World!", b"Old World!", "Old World!"))
+@pytest.mark.parametrize("req_file", (b"Old World!", "Old World!", b"Old World!"))
```

Comment:
Should the test have a non utf8 byte in it?

Reply:
@markstory 
of course, added into matrix